RefrigeratorExe/RefrigeratorExe/Item.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
```

Comment:
It is a good practice not to shorten the names. Next time call it RefrigiratorExcersize or RefrigiratorManager or simply Refrgirator